pkgs/development/python-modules/xmodem/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, buildPythonPackage, fetchPypi, tox }:
+
+buildPythonPackage rec {
+  pname = "xmodem";
+  version = "0.4.5";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "8b7ca0f45cdf0f5e7212a60931c98b333da5ef2fc6702a5d10c51b9fc52ce91c";
+  };
+
+  # Tests are deprecated and zero tests are run
+  doCheck = false;
```

Comment:
it's likely that their source distribution doesn't have tests. It would be preferable to checkout from github so that we can get some unit tests.

Reply:
Correct, thanks. I amended the package to fetch from github and added the necessary `buildInputs`. Tests not succeed.